demo/demo.html

```diff
@@ -49,7 +49,7 @@ <h1 class="text-muted">&lt;json-formatter&gt;</h1>
       <p>Use <code>&lt;json-formatter&gt;</code> directive</p>
       <p>
         <pre>
-          &lt;json-formatter open="1" json="{my: 'json'}"&gt;&lt;/json-formatter&gt;
+          &lt;json-formatter open="1" json="{my: 'json'}" thumbnail="1"&gt;&lt;/json-formatter&gt;
```

Comment:
Please use a "provider" for this configuration. This is something users will set per app.

See an example of exposing a provider here https://github.com/mohsen1/angular-json-schema-form/blob/master/src/schema-form.js#L103